Index: Resources.Designer.cs
===================================================================
--- Resources.Designer.cs	(revision 324)
+++ Resources.Designer.cs	(working copy)
@@ -39,7 +39,7 @@
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("LanExchange.Properties.Resources", typeof(Resources).Assembly);
+                    global::System.Resources.ResourceManager temp = new SDK.TranslationResourceManager("LanExchange.Properties.Resources", typeof(Resources).Assembly);
                     resourceMan = temp;
                 }
                 return resourceMan;
